app/helpers/application_helper.rb

```diff
@@ -67,10 +62,11 @@ def format_date(date, format = :long)
     date && l(date.to_date, format: format) || '-'
   end
 
+  ## Conferences
   def format_conference_date(starts_on, ends_on)
-    return unless [starts_on, ends_on].all?(&:present?)
-    starts_on = starts_on.strftime('%d %b %y')
-    ends_on = ends_on.strftime('%d %b %y')
+    return unless starts_on && ends_on
```

Comment:
Is this a scenario that could ever happen? As far as I understand it, there are presence validations on these fields already...?